### DISCUSSION TITLE
Simplelogin indexed by hunter.io (solution in the post)

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1529] [Q&A] [answered]
Hello!
Service hunter.io indexes your domains (ALL paid and free) (screenshot 1), when the domain protonmail.com is excluded from the search because of that is a personal email creating service (screenshot 2).

Please contact the hunter.io team with a request to exclude your domains from the search as well. **They exclude all domains that are using to create personal addresses (but they need an official request).**

[Screenshot 1](https://user-images.githubusercontent.com/122488962/211918225-897cd56a-5e00-4f18-ab4d-8b9a3c5aefd8.jpeg) 
[Screenshot 2](https://user-images.githubusercontent.com/122488962/211918247-5d8420d9-dcab-41be-b73e-1a4189fc2e3f.jpeg)

**Comment by 4b15ed24f576** [comment]
Hi there, thank you for your report and for bringing this to our attention.

We contacted the team at hunter.io and ask them to remove our domains from their service.

**Comment by AmrithVengalath** [comment, marked as answer]
This issue is solved by hunter team. Here is [the link to their reply](https://github.com/ProtonMail/WebClients/issues/300#issuecomment-1429283535)

**Reply by nguyenkims** [reply]
That's great, thanks!
